benchmark-e2e/src/main/java/io/opentelemetry/e2ebenchmark/E2EAgentBenchmark.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.e2ebenchmark;
+
+import java.util.AbstractMap;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.lifecycle.Startables;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+public class E2EAgentBenchmark {
+  private String app = System.getenv("APP_IMAGE");
+
+  private static final Logger LOG = LoggerFactory.getLogger(E2EAgentBenchmark.class);
+  private List<GenericContainer<?>> containers;
+
+  @BeforeEach
+  void setUp() {
+    containers = new ArrayList<>();
+  }
+
+  @AfterEach
+  void tearDown() {
+    containers.forEach(GenericContainer::stop);
+  }
+
+  @Test
+  void run() throws Exception {
+    runBenchmark();
+  }
+
+  private void runBenchmark() throws Exception {
+    if (app == null || app.equals("")) {
+      // setting default image to benchmark
+      app = "ghcr.io/open-telemetry/java-test-containers:smoke-springboot-jdk8-20201204.400701583";
+    }
+
+    String agentPath = System.getProperty("io.opentelemetry.smoketest.agent.shadowJar.path");
+
+    // docker images
+    final DockerImageName WRK_IMAGE = DockerImageName.parse("quay.io/dim/wrk:stable");
+    final DockerImageName APP_IMAGE = DockerImageName.parse(app);
+    final DockerImageName OTLP_COLLECTOR_IMAGE =
+        DockerImageName.parse("otel/opentelemetry-collector-dev:latest");
```

Comment:
it looks like checkstyle is now complaining about these, can u make them camelCase? that should resolve

Reply:
I'm ok with loosening the checkstyle actually - I don't see why an abbreviation would need to be limitted to just 2 letters. (seems to not like WRK being 3 letters, APP, OTLP)

Reply:
it's complaining b/c those are using constant style naming `APP` instead of instance/local var naming `app`, which goes against https://google.github.io/styleguide/javaguide.html

Reply:
if you did `static final DockerImageName APP_IMAGE = DockerImageName.parse(app);` checkstyle wouldn't complain

Reply:
the "abbreviation" thing is red-herring, it's to allow something like "serverURL" if you like that instead of "serverUrl"

Reply:
Oh duh - guess these should be static :)